QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -40,5 +42,6 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
 		waitForTime(timeout);
 		
 		Rollers.running.set(false);
+		done.set(true);
```

Comment:
What about if there's an error? Make sure that done gets set back to true. Or at least report something.